hedyorg/hedy
# [CONTENT] Fixes to Level 8 Quiz

**Description**

Fixes some grammatical issues with the level 8 quiz. In question 4, "Were going on holiday!" was changed to "We are going on vacation!". In question 5, "last two lines should start on new lines en start with 4 spaces" is changed to 
"last two lines should start on new lines and start with 4 spaces". Lastly, in question 7, the correct variable name that is used in the problem was "food," however, the first line referred to it as "eten."



**Fixes _issue or discussion number_**

Fixes #3506


**How to test**

If you have pulled my code, go to "/quiz/preview-question/8/3", "/quiz/preview-question/8/4", "/quiz/preview-question/8/6", you can see the changes. Note, there is a bug in the "/preview-question/<level>/<question_num>" route, where it runs question_num + 1. 

question 4 - 

<img width="1218" alt="Screen Shot 2022-11-10 at 4 39 43 PM" src="https://user-images.githubusercontent.com/59891012/201211446-65e533a5-94ef-4eb2-b8ec-4bfac6a06ed9.png">


question 5 - 
<img width="601" alt="Screen Shot 2022-11-10 at 4 39 26 PM" src="https://user-images.githubusercontent.com/59891012/201211401-cf500bc1-c4a4-41d4-8974-24a805347f85.png">


 question 7 -

<img width="983" alt="Screen Shot 2022-11-10 at 4 38 03 PM" src="https://user-images.githubusercontent.com/59891012/201211198-53437eb1-8625-4c2c-8c1d-f9aa64955a70.png">

**Checklist**
Done? Check if you have it all in place using this list:*
  

If you're unsure about any of these, don't hesitate to ask. We're here to help!
